templates/unifi-poller.xml

```diff
@@ -31,110 +29,105 @@ https://github.com/davidnewhall/unifi-poller/wiki/Installation</Description>
-  <Config Name="InfluxDB Password" Target="UP_INFLUX_PASS" Default="unraid" Mode="" Description="Container Variable: UP_INFLUX_PASS&#13;&#10;- Password for InflluxDB (if required)" Type="Variable" Display="always-hide" Required="false" Mask="true"/>
-  <Config Name="HTTP Listen Address" Target="UP_HTTP_LISTEN" Default="0.0.0.0:9031" Mode="" Description="Container Value: UP_HTTP_LISTEN&#13;&#10;- Web port poller listens while in prometheus (or both) MODE. Must be in 0.0.0.0:port format." Type="Variable" Display="advanced-hide" Required="false" Mask="false"/>
-  <Config Name="Poll Sites" Target="UP_SITES" Default="all" Mode="" Description="Container Variable: UP_SITES&#13;&#10;- List of sites to poll on controller, comma separated. Use cryptic ID values here." Type="Variable" Display="advanced-hide" Required="false" Mask="false"/>
-  <Config Name="HTTP Port" Target="9031" Default="9031" Mode="tcp" Description="This needs to match UP_HTTP_LISTEN - this is the port prometheus connects to to pick up metrics." Type="Port" Display="advanced-hide" Required="false" Mask="false"/>
-  <Config Name="Save IDS" Target="UP_SAVE_IDS" Default="false" Mode="" Description="Container Variable: UP_SAVE_IDS&#13;&#10;- Save intrusion detection data in InfluxDB. This data does not contain graphable metrics." Type="Variable" Display="advanced-hide" Required="false" Mask="false"/>
-  <Config Name="Save Sites" Target="UP_SAVE_SITES" Default="true" Mode="" Description="Container Value: UP_SAVE_SITES&#13;&#10;- Disable this to save space and resources in InfluxBD. This data powers the Network Sites dashboard." Type="Variable" Display="advanced-hide" Required="false" Mask="false"/>
-  <Config Name="UniFi Re-authenticate" Target="UP_REAUTHENTICATE" Default="false" Mode="" Description="Container Variable: UP_REAUTHENTICATE&#13;&#10;- This is a debug method. Do not enable unless instructed to do so." Type="Variable" Display="advanced-hide" Required="false" Mask="false"/>
-  <Config Name="UniFi Verify SSL" Target="UP_VERIFY_SSL" Default="false" Mode="" Description="Container Variable: UP_VERIFY_SSL&#13;&#10;- Enable this if you have a valid SSL cert on your controller." Type="Variable" Display="advanced-hide" Required="false" Mask="false"/>
-  <Config Name="Quiet mode" Target="UP_QUIET" Default="false" Mode="" Description="Container Variable: UP_QUIET&#13;&#10;- Disables most log output." Type="Variable" Display="advanced-hide" Required="false" Mask="false"/>
+  <Labels/>
+  <!-- poller -->
+  <Config Name="Poller Debug Mode" Target="UP_POLLER_DEBUG" Default="true|false" Mode="" Description="Container Variable: UP_POLLER_DEBUG - Setting true includes more log output." Type="Variable" Display="always-hide" Required="false" Mask="false">false</Config>
```

Comment:
The bool wont work, because a value is already set.

Reply:
Does it fail silently? It seemed to work..... I'll check again!

Reply:
in reality you are just choosing for the user. guess thats fine

Reply:
I played with this. If you add `true` or `false` between `<Config>` and `</Config>` it takes that as the default value. You can still change it. If you leave the value missing from between the open and close tag it takes the first value inside the `Default` param. For example you can set a default of `true` by passing `Default="true|false"` and you can set a default of `false` with `Default="false|true"`. I went ahead and did it this second way.